lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -979,10 +979,14 @@ previewers.git_file_diff = defaulter(function(opts)
         if p == nil or p == "" then
           return
         end
-        conf.buffer_previewer_maker(p, self.state.bufnr, {
-          bufname = self.state.bufname,
-          winid = self.state.winid,
-        })
+        conf.buffer_previewer_maker(
+          p,
+          self.state.bufnr,
+          vim.tbl_extend("force", opts, {
+            bufname = self.state.bufname,
+            winid = self.state.winid,
```

Comment:
hmm we actually just need

```lua
        preview = opts.preview,
        file_encoding = opts.file_encoding,
```

i dont think we wanna merge this.

Reply:
Hi. Thank you for the comment.
I followed your codes.
Does it makes you wanna merge?
If not, I appreciate if you would tell me why.